projects/components/src/load-async/load-async.directive.ts

```diff
@@ -23,8 +24,11 @@ import {
 export class LoadAsyncDirective implements OnChanges, OnDestroy {
   @Input('htLoadAsync')
   public data$?: Observable<unknown>;
+  @Input()
+  public htLoadAsyncLoaderType!: LoaderType;
```

Comment:
this should be optional. You can assign a default value here

Reply:
updated